doc/zh-cn/plugins/consumer-restriction.md

```diff
@@ -20,27 +20,36 @@
 - [English](../../plugins/consumer-restriction.md)
 
 # 目录
-- [**名字**](#名字)
-- [**属性**](#属性)
-- [**如何启用**](#如何启用)
-- [**测试插件**](#测试插件)
-- [**禁用插件**](#禁用插件)
+- [目录](#目录)
+  - [名字](#名字)
+  - [属性](#属性)
+  - [如何启用 `consumer_name`](#如何启用-consumer_name)
+  - [测试插件](#测试插件)
+  - [如何限制 service ID](#如何限制-service-id)
+  - [路由测试](#路由测试)
+  - [路由测试](#路由测试-1)
+  - [禁用插件](#禁用插件)
 
 ## 名字
 
-`consumer-restriction` 可以通过以下方式限制对服务或路线的访问，将 consumer 列入白名单或黑名单。 支持单个或多个 consumer。
+`consumer-restriction` 根据选择的不同对象做相应的访问限制，支持 consumer name 和 service id 两种限制类型。
 
 ## 属性
 
-* `whitelist`: 可选，加入白名单的consumer
-* `blacklist`: 可选，加入黑名单的consumer
+* consumer name：把 `consumer` 的 `username` 列入白名单或黑名单（支持单个或多个 consumer）来限制对服务或路线的访问。
+* service id：把 `service` 的 `id` 列入白名单或黑名单（支持一个或多个 service）来限制service的访问，需要结合授权插件一起使用。
```

Comment:
Why are these two separately listed outside?

Reply:
The "service id" method needs to be used in conjunction with authorized plugins. It is not easy to confuse the use scene after separating them, I think it will be better.